### ISSUE TITLE
Blossom flower?

### ISSUE THREAD

**Issue by chimericdream** [frikinjay/morefrogs#18]
In my modpack, we have the "Random Bone Meal Flowers" mod installed, which allows for any flower to spawn when using bone meal on grass. I noticed that a flower with the tooltip `block.morefrogs.blossom_flower` is spawning with a broken texture, but I didn't see that in REI. Is this an unfinished feature that we should disable in the bone meal mod's config for now?

**Comment by frikinjay** [comment]
Yes, this was/is a WIP feature that somehow made it into a release, you can turn it off.
